package.json

```diff
@@ -26,7 +26,8 @@
     "test:watch": "jest app/javascript/ bin/ --watch",
     "postcss": "postcss public/assets/*.css -d public/assets",
     "e2e": "bin/e2e",
-    "e2e:creator-onboarding-seed": "yarn run e2e --creator-onboarding-seed"
+    "e2e:creator-onboarding-seed": "yarn run e2e --creator-onboarding-seed",
+    "build": "./node_modules/.bin/esbuild app/javascript/*.* --bundle --loader:.erb=file --loader:.toml=file --sourcemap --outdir=app/assets/builds --public-path=assets"
```

Comment:
added the `--loader` options to use the `file` loader on the non-JS files. https://esbuild.github.io/content-types/#file not sure if there's a more terse way to express this, but these additions got me past the "no loader is configured" errors I'd been getting. Currently working on verifying things work in Canary.